react/src/components/atoms/forms/InputNumber/index.js

```diff
@@ -52,45 +52,34 @@ const NumberInput = (props) => {
               errorMsg: ''
             };
           };
-          const hasProperty = (obj, property) => Object.prototype.hasOwnProperty.call(obj, property) && !is.nil(obj[property]);
+
+          const hasNumberProperty = (obj, property) => Object.prototype.hasOwnProperty.call(obj, property) && is.number(obj[property]);
 
           const handleOnBlur = (e) => {
             e.persist();
             const inputEl = ref.current;
-            let newValue = Number(inputEl.value);
-            if ((hasProperty(props, 'max') && newValue > props.max) || (hasProperty(props, 'min') && newValue < props.min)) {
-              if (hasProperty(props, 'max') && newValue > props.max) {
-                newValue = props.max;
-              }
-              if (hasProperty(props, 'min') && newValue < props.min) {
-                newValue = props.min;
-              }
+            let newValue = inputEl.value ? Number(inputEl.value) : inputEl.value;
+            if (hasNumberProperty(props, 'max') && newValue > props.max) {
```

Comment:
```suggestion
            if (hasNumberProperty(props, 'max') && (is.number(newValue) && newValue > props.max)) {
```

Reply:
check already happend --> [on line 61](https://github.com/massgov/mayflower/pull/518/files#diff-6eb9a368ca77d1f2523e456949ebae6fR61)

Reply:
this check is not needed, you can only enter numbers, periods, commas, or null in an input type `number`, so this will be a number null, or a string of `.` or `,` which will evaluate to false above.